docs/UserGuide.md

```diff
@@ -3,14 +3,141 @@ layout: page
 title: User Guide
 ---
 
+# Table of Contents
+<!-- TOC -->
+* [Introduction](#introduction)
+  * [About TaskHub](#about-taskhub)
+  * [About this User Guide](#about-this-user-guide)
+  * [Navigating the User Guide](#navigating-the-user-guide)
+* [Getting Started](#getting-started)
+  * [Installing and launching TaskHub](#installing-and-launching-taskhub)
+  * [Understanding the components of TaskHub](#understanding-the-components-of-taskhub)
+  * [Understanding the User Interface(UI)](#understanding-the-user-interface--ui-)
+  * [Quick start](#quick-start)
+* [Features](#features)
+  * [General Commands](#general-commands)
+    * [View help : `help`](#view-help--help)
+      * [GUI Help:](#gui-help-)
+      * [CLI Help](#cli-help)
+    * [List all employees and projects: `list`](#list-all-employees-and-projects--list)
+    * [Clear all entries : `clear`](#clear-all-entries--clear)
+    * [Exit the program : `exit`](#exit-the-program--exit)
+  * [Employee Commands](#employee-commands)
+    * [Add an employee: `addE`](#add-an-employee--adde)
+    * [Edit an employee : `editE`](#edit-an-employee--edite)
+    * [Delete an employee : `deleteE`](#delete-an-employee--deletee)
+    * [List all employees : `listE`](#list-all-employees--liste)
+    * [Locate employees by name: `findE`](#locate-employees-by-name--finde)
+  * [Project Commands](#project-commands)
+    * [Add a new project : `addP`](#add-a-new-project--addp)
+    * [Edit a project : `editP`](#edit-a-project--editp)
```

Comment:
Seems like some of the above lines have an extra space before the colon!

Reply:
It seems that some of the commands (in the features section) had these spaces and it carried over when the table of contents was generated. I'll also standardise the spacing for the commands.